test/smoke/src/sql/main.ts

```diff
@@ -29,7 +29,7 @@ const PLATFORM = '${PLATFORM}';
 const RUNTIME = '${RUNTIME}';
 const VERSION = '${VERSION}';
 
-const sqliteUrl = `https://github.com/kburtram/azuredatastudio-sqlite/releases/download/1.0.20/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
+const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/1.0.0/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
```

Comment:
should we use a lower version here?  typically that is an anti-pattern.

Reply:
I restarted it at 1.0.0 since I didn't copy over any of the history - so essentially this is a completely new thing.

But I don't see any reason not to bump it up and avoid any possibility of confusion so yeah I'll go update it.